storybook/src/server/framework-preset.ts

```diff
@@ -17,36 +14,39 @@ import dedent from 'ts-dedent';
 type BuildOptions = {
     twigComponent: TwigComponentConfiguration;
     twig: TwigConfiguration;
-    runtimeDir: string;
-    projectDir: string;
     additionalWatchPaths: string[];
+    templatePathAliases: {
+        [p: string]: string;
+    };
 };
 
 const getBuildOptions = async (symfonyOptions: SymfonyOptions) => {
-    const projectDir = await getKernelProjectDir();
-    const twigComponentsConfig = await getTwigComponentConfiguration();
-    const twigConfig = await getTwigConfiguration();
+    const symfonyConfig = await getSymfonyConfig(symfonyOptions.storybookCachePath);
 
     const componentNamespaces: { [p: string]: string[] } = {};
 
-    const twigPaths: string[] = Object.keys(twigConfig.paths).map((key) => `${projectDir}/${key}/`);
+    const twigPaths: string[] = Object.keys(symfonyConfig.twig_config.paths).map((key) => {
+        if (key.startsWith(symfonyOptions.projectDir)) {
+            return `${key}/`;
+        }
+
+        return `${symfonyOptions.projectDir}/${key}/`;
+    });
```

Comment:
I don't know if I misconfigured my Symfony app, but values from `symfonyConfig.twig_config.paths` already contains `symfonyOptions.projectDir`

Without this change, path is duplicated

Here is my `twig.yaml`

```yaml
twig:
    file_name_pattern: '*.twig'
    default_path: '%kernel.project_dir%/templates'
    paths:
        '%kernel.project_dir%/templates': Shared
```